### PR TITLE
feat(website): blocks implementation

### PR DESCRIPTION
**Pull request by Itrulia**
ToDos:
- [x] ImageBlock: Implement Image srcset
- [x] FlexGridTeaser: Add Teaser to website builder

re WPC-1036, WPC-1075, WPC-1062, WPC-1063, WPC-1071, WPC-1066, WPC-1072

### CODE REVIEW

File: libs/image/website/src/lib/image.tsx [hunk 1]
```diff
@@ -0,0 +1,56 @@
+import {styled} from '@mui/material'
+import {FullImageFragment} from '@wepublish/website/api'
+import {BuilderImageProps} from '@wepublish/website/builder'
+
+declare module 'react' {
```
Comment (Itrulia) [inline libs/image/website/src/lib/image.tsx:5]: missing in the react typings

File: libs/website/builder/src/lib/website-builder.context.tsx [hunk 2]
```diff
@@ -141,6 +147,8 @@ export const WebsiteBuilderProvider = memo<
   const parentComponents = useWebsiteBuilder()
   const newComponents = mergeDeepRight(parentComponents, components) as WebsiteBuilderComponents
 
+  console.log(newComponents)
```
Comment (penk85) [inline libs/website/builder/src/lib/website-builder.context.tsx:150]: You want this here?

**Review by penk85 — DISMISSED** [review]

File: libs/navigation/website/src/lib/footer/__snapshots__/footer-container.spec.tsx.snap [hunk 3]
```diff
@@ -24,7 +24,23 @@ exports[`Footer Container should render Default 1`] = `
       </svg>
```
Comment (antkiewiczk) [inline libs/navigation/website/src/lib/footer/__snapshots__/footer-container.spec.tsx.snap:24]: navbar and footer are implemented in another PR, that's why here there are only tests?

File: libs/website/api/src/lib/schemas/image.graphql [hunk 4]
```diff
@@ -1,12 +1,15 @@
...
   url
-  largeURL: transformURL(input: {width: 500})
-  mediumURL: transformURL(input: {width: 300})
-  thumbURL: transformURL(input: {width: 280, height: 200})
-  squareURL: transformURL(input: {width: 100, height: 100})
-  previewURL: transformURL(input: {width: 400, height: 200})
-  column1URL: transformURL(input: {width: 800, height: 300})
-  column6URL: transformURL(input: {width: 260, height: 300})
+
+  bigURL: transformURL(input: {width: 800, output: WEBP, quality: 1})
```
Comment (antkiewiczk) [inline libs/website/api/src/lib/schemas/image.graphql:4]: why these changes? `output: WEBP, quality: 1` looks reasonable, but won't changing names like `bigURL` break anything?

File: tsconfig.base.json [hunk 5]
```diff
@@ -24,12 +24,14 @@
...
+      "@wepublish/consent/api": ["libs/consent/api/src/index.ts"],
+      "@wepublish/consent/editor": ["libs/consent/editor/src/index.ts"],
       "@wepublish/editor/api": ["libs/editor/api/src/index.ts"],
       "@wepublish/editor/api-v2": ["libs/editor/api-v2/src/index.ts"],
       "@wepublish/event/website": ["libs/event/website/src/index.ts"],
-      "@wepublish/consent/api": ["libs/consent/api/src/index.ts"],
-      "@wepublish/consent/editor": ["libs/consent/editor/src/index.ts"],
       "@wepublish/image/api": ["libs/image/api/src/index.ts"],
+      "@wepublish/image/website": ["libs/image/website/src/index.ts"],
+      "@wepublish/image/website/server": ["libs/image/website/src/server.ts"],
```
Comment (antkiewiczk) [inline tsconfig.base.json:34]: can't see this file 🤔

**Review by antkiewiczk — COMMENTED** [review]
one more thing: are we planning to deploy the storybook with PR so that it's possible to see the ui? I think that would be useful

File: libs/navigation/website/src/lib/footer/__snapshots__/footer-container.spec.tsx.snap (hunk 3 above)
Comment (Itrulia) [reply libs/navigation/website/src/lib/footer/__snapshots__/footer-container.spec.tsx.snap:24]: It's because I fixed the mock, so they showed up :D

**Comment by Itrulia** [comment]
@antkiewiczk yeah storybook deployments will come. Fixed the things you found

File: libs/website/builder/src/lib/website-builder.context.tsx (hunk 2 above)
Comment (Itrulia) [reply libs/website/builder/src/lib/website-builder.context.tsx:150]: https://github.com/wepublish/wepublish/pull/942/files#diff-60aa3fdb259a650b1fab2bf6dcb3ff07ae1a923b6e3da03beb040c72cf354a6a

i'll be removing this here, is it fine if I merge them together?

**Review by antkiewiczk — DISMISSED** [review]
